kuma/static/styles/zones/components/articulated-diagram.scss

```diff
@@ -1,10 +0,0 @@
-.articulated-diagram-component {
```

Comment:
You'll need to remove the ``@import`` of this file in ``kuma/static/styles/zones.scss``.